bin/mozetl-databricks.py

```diff
@@ -127,6 +199,13 @@ def parse_arguments():
         required=True,
         help="A Databricks authorization token, generated from the user settings page",
     )
+    parser.add_argument(
+        "--pypi-libs",
+        type=str,
+        default="",
+        help="PyPI libraries to install",
+        action=coerce_pypi_names("pypi_libs"),
```

Comment:
Small example in the help string would be helpful.

Reply:
Added an example in 8c56f33